README.md

```diff
@@ -68,53 +70,51 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
-		  branch: newTranslations
-		  author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+          branch: newTranslations
+          author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
           commit-message: Updated translations
           title: Updated translations 
           body: Pulled in new translations from Applanga portal
           path: 'checkout'
 ```
 
+
 ---
 ## Configure Webhook Endpoint
-To the trigger the `applanga-pull` workflow a webhook endpoint has to configured in the projects settings page on applanga. 
+To the trigger the `applanga-pull` workflow a webhook endpoint has to be configured in the projects settings page on applanga. 
 
 The webhook is triggered at least 15 minutes from when there is no translation change. This means whenever translation is added or edited a webhook request is scheduled to be sent to all configured endpoints for the project 15 minutes later. The scheduled request will be sent as planned unless there is a new change to translation before the scheduled time, in which case it is rescheduled to be sent 15 minutes later.
 
-The endpoint should be configured to match the following `Github REST API` post request:
-
-```shell
-curl -v -H "Accept: application/vnd.github.everest-preview+json" \
-        -H "Authorization: token $PERSONAL_ACCESS_TOKEN" \
-        https://api.github.com/repos/:owner/:repo/dispatches \
-        -d '{"event_type":"applanga-pull"}'
-```
-
 Here are the steps to setup the **Webhook Endpoint**
 * Login to Applanga dashboard and navigate to the project. Then click on **Project Settings**
 
 ![](https://www.applanga.com/assets/images/docu/groups_editapp.png)
 
-* In the settings page scroll down to the section **WEB HOOKS** and click the **Add endpoint** button, this will show a modal
-where the endpoint values can be entered
+* In the settings page scroll down to the section **WEB HOOKS** and click the **Add endpoint** button, this will show a modal where the endpoint values can be entered
 ![](https://www.applanga.com/assets/images/docu/webhook_settings.png)
 
-The values should be configured to match the preceding curl request. To be sure, verify the following
-- **POST** is selected as http method
-- **JSON** is selected in the body tab
-- `{"event_type":"applanga-pull"}` is entered in the **Request body**
+* Set the http method to *POST* and enter the endpoint url as follows `https://api.github.com/repos/<OWNER>/<REPO>/actions/workflows/<WORKFLOW_ID>`. The following values `<OWNER>`, `<REPO>` and `<WORKFLOW_ID>` should be replaced with the correct values. The `WORKFLOW_ID` should be the name of the workflow file you want to trigger, in this case `applanga-pull.yml`.
```

Comment:
no need to refer to WORKFLOW_ID just put the correct filename here applanga-pull.yml

Reply:
fixed